core/templates/pages/topic-editor-page/modal-templates/preview-thumbnail.directive.html

```diff
@@ -0,0 +1,54 @@
+<div class="oppia-thumbnail-container protractor-test-thumbnail-container" ng-style="{background: '#2F6687', color: '#FFFFFF', width: '248px' }">
```

Comment:
Just use a class if the style is not expected to change dynamically.

Reply:
Yeah right. Done